docs/devices/TS004F.md

```diff
@@ -28,6 +28,26 @@ pageClass: device-page
 <!-- Notes END: Do not edit below this line -->
```

Comment:
- Please add everything above this line
- I'm not sure if the serial number is useful information, this device will probably come with many different serial numbers since the module is used in a lot of devices.

Reply:
I added the serial because the inside differs from the outside. On the TS0044 page the serial is also mentioned. Maybe only mention the inside serial?

Reply:
I see a typo in the serial number header..